.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please feel free to remove whatever sections/lines in this aren't relevant. -->
+
+## Title Line Template: [Website] - [UI components]: [Brief description.]
+<!-- Use the title line as the title of your pull request, then delete these lines. 
+
+Website: Issues that impact standards.usa.gov look, feel or functionality.
+UI components: Issues that impact the look, feel or functionality of the standards themselves.
+
+-->
```

Comment:
Using `<!-- ... -->`  doesn't do anything in the markdown file. Maybe this should be marked up in another way, like the quote style, i.e. `>Some text...`